scenarios/centos-9/edpm_baremetal_deployment_ci.yml

```diff
@@ -6,7 +6,8 @@ cifmw_install_yamls_vars:
   STORAGE_CLASS: crc-csi-hostpath-provisioner
   DATAPLANE_REPO: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/dataplane-operator"
   INFRA_REPO: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/infra-operator"
-  BMAAS_INSTANCE_MEMORY: 6144
+  BMAAS_INSTANCE_MEMORY: 8192
+  BMAAS_INSTANCE_VCPUS: 6
```

Comment:
We probaly should increase the storage for the edpm node too

Reply:
Now the job is failing becauase they're trying to download container images before growvol runs.

Reply:
The default size of edpm node is 20GB as per https://github.com/openstack-k8s-operators/install_yamls/blob/main/devsetup/Makefile#L12

so, I making it 40GB on edpm node, please let me know if we need to bump it further. The VM on which job is running have 400GB disk.

Reply:
40 worked for me locally.

Reply:
I've also proposed to move download-cache service after configure-network. https://github.com/openstack-k8s-operators/dataplane-operator/pull/429

Reply:
thanks!